pages/careers/_open-positions.html

```diff
@@ -237,5 +320,267 @@ <h4>Help us a create a fundamentally better DevOps experience!</h4>
+      </li>
+      <li>
+        <strong>Compliance.</strong>
+        Finally, you'll also be the owner of
+        <a href="achieve-compliance" target="_blank">Gruntwork Compliance</a>, which includes out-of-the-box compliance
+        with the CIS AWS Foundations Benchmark today, and soon, SOC 2. You'll set the direction for our compliance
+        business, including figuring out which standards we'll target in the future (e.g., ISO 27001, PCI, HIPAA, other
+        CIS benchmarks, etc.), how to package the product, how to help customers ensure ongoing compliance, and so on.
+      </li>
+      <li>
+        <strong>Work alongside engineering and design.</strong>
+        You'll serve as one member of an executive "trio" that includes an engineering manager and product designer.
+        You'll collaborate with your colleagues on the vision, usability, and business viability of this new product,
+        and take primary responsibility for our ability to execute. Further, you'll work directly with and report to
+        one of our co-founders.
+      </li>
+      <li>
+        <strong>Work alongside customers.</strong>
+        You'll work directly with our customers to understand what problems they are facing, figure out solutions to
+        those problems, guide the team in building those solutions, share those solutions with customers, gather
+        feedback, and iterate.
+      </li>
+      <li>
+        <strong>Drive our product management practices.</strong>
+        Help improve how we build products as a company by streamlining all of our product management processes,
+        including product planning, user testing, user story mapping, task tracking, backlog grooming, sprint planning
+        and execution, market research, pricing strategy, customer communication, and more.
+      </li>
+    </ul>
+    <h4>Your Ideal Background</h4>
+    <ul>
+      <li>
+        <strong>Product sensibility.</strong>
+        As the future owner of several major new products at Gruntwork, we'll look to you for leadership, insights, and
+        methodologies that will help us translate our shared vision into a fantastic experience for customers.
+      </li>
+      <li>
+        <strong>Leadership.</strong>
+        We're looking for someone with a proven track record of delivering products that move the needle. You must
+        have the ability to lead a team through ideation, prototyping, user testing, development, launch, and iteration.
+      </li>
+      <li>
+        <strong>Domain expertise.</strong>
+        We don't expect you to know everything coming in, but to create products in our space, you must be able to
+        quickly build a deep understanding of our domain: DevOps, cloud software, software delivery, CI / CD, and
+        infrastructure as code. If you happen to have prior experience in software engineering, working on dev tools, o
+        other technical domains, that is a big plus.
+      </li>
```

Comment:
I think we should prior experience as a software engineer or a degree in computer science a minimum, not just something we hope they "happen" to have. There are lots of product managers I've seen who began as software engineers, and that seems like a key requirement to me.

Reply:
That's a good point. Updated to required: https://github.com/gruntwork-io/gruntwork-io.github.io/pull/710/commits/433e55ea1f82d3d8ebee173a7009f1b5f4f1fd22